projects/seeker/agents/seeker.py

```diff
@@ -201,7 +201,7 @@ def add_additional_subagent_args(cls, parser: ParlaiParser) -> ParlaiParser:
         """
         additional_agent_parser = cls.get_additional_agent_args()
         for action in additional_agent_parser._actions:
-            key = action.option_strings[-1]
+            key = sorted(action.option_strings, key=lambda x: len(x))[-1]
```

Comment:
nit: do you think using `max` makes it slightly shorter? You wouldn't need the `[-1]`.

Reply:
oh yeah good point

Reply:
since we're dealing with strings not numbers

Reply:
I don't understand, the behavior shouldn't change between the max or sort here.